org.blackfennec.extensions.core.yml

```
@@ -0,0 +1,18 @@
+id: org.blackfennec.extensions.core
```

```suggestion
id: org.blackfennec.app.extensions.core
```

Extensions targeting single app should use that app's ID.

---

we would have named the app `org.blackfennec` if we could have :grinning:  
i personally prefer the version without the `app`. 

Is it a SHOULD or a COULD?
Consider it done iff its the former :smile:

---

Let me rephrase it: they need to match.

---

> we would have named the app `org.blackfennec` if we could have 😀
> i personally prefer the version without the `app`.

Thing is usually I outline that `app` isn't proper (I would have suggested `blackfennec` or anything that isn't `app` or `desktop`) but it appear that I didn't this time.

---

And just to clarify `org.blackfennec` would be rejected flat out.